code/modules/mob/living/carbon/human/human.dm

```diff
@@ -228,12 +231,12 @@
 				throw_at(target, 200, 4)
 
 				var/limbs_affected = pick(2,3,4)
-				var/datum/organ/external/processing_dismember
+				var/obj/item/organ/external/processing_dismember
 
 				while(limbs_affected != 0)
 					processing_dismember = pick(organs)
-					if(processing_dismember.name != "chest" && processing_dismember.name != "head")
-						processing_dismember.droplimb(1,0,1,pick(0,1,2))
+					if(processing_dismember.name != "chest" && processing_dismember.name != "head" && processing_dismember.name != "groin")
+						processing_dismember.droplimb(1,pick(0,1,2))

```

Comment:
If the "nodamage" var for droplimb is implemented below, then explosion code would be the ideal place to utilize this, since explosion code inherently bypasses the max_damage requirements to dismember--since it does so, this will make explosions far far more devastating than they currently are as the dismemberment will cause extra damage in addition to the regular explosion damage.